Examples/doc-example-apis/Sources/main.swift

```diff
@@ -0,0 +1,201 @@
+import Momento
+import Foundation
+
+@available(macOS 10.15, iOS 13, *)
+func example_API_InstantiateCacheClient() {
+    do {
+        let credentialProvider = try CredentialProvider.fromEnvironmentVariable(envVariableName: "MOMENTO_API_KEY")
+        let cacheClient = CacheClient(
+            configuration: CacheClientConfigurations.iOS.latest(),
+            credentialProvider: credentialProvider,
+            defaultTtlSeconds: 10
+        )
+    } catch {
+        print("Unable to create cache client: \(error)")
```

Comment:
To use this with CI to detect failures, we need to go ahead and exit the program with a non-zero exit code any time we catch an error like this or get an error response back from an API call in the other methods. Adding `exit(1)` below the `print` here will tell CI that we have a failure.